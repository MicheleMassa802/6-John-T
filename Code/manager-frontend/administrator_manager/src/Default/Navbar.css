.navbar {
    background-color: rgb(196, 181, 155);
    height: 100px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: left;
    padding: 10px;
    box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
  }
  
  .navbar-title {
    margin: 0;
    font-size: 24px;
    color: #333;
  }
  
  .navbar-items {
    display: inline-flex;
    justify-content: space-between;
    align-items: center;
    margin-top: 10px;
    
  }
  
  .navbar-item {
    margin: 5px;
    font-size: 20px;
    color: #333;
    text-decoration: none;
    padding: 10px;
    background-color: rgb(196, 181, 155);
    border: 1px solid rgb(96, 81, 55);
    border-radius: 5px;
    box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
  }