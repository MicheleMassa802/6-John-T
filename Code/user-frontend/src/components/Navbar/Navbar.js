import { Link, Outlet } from "react-router-dom";
import logo from "../../assets/logo.png";
import React from 'react';

const Navbar = () => {

    return (
        <>
            <nav className="nav"
                style={{
                    position: "fixed",
                    top: "0px",
                    width: "100%",
                    display: "flex",
                    height: "80px",
                    "box-shadow": "0 0.1vw 0.5vw 0 #00000066",
                    "z-index": 99999,
                    "background-color": "#fff",
                    "align-items": "center"
                }}
            >
                <Link to="/">
                    <img className="logo-main"
                        src={logo}
                        alt=""
                        style={{
                            display: "block",
                            "margin-left": "auto",
                            "margin-right": "auto",
                            height: "50px",
                            "z-index": 999999
                        }} />
                </Link>
            </nav>
            <Outlet />
        </>
    )
}

export default Navbar;