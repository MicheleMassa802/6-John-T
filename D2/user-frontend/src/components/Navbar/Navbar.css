.logo-main {
    display: block;
    margin-left: auto;
    margin-right: auto;
    height: 50px;
    z-index: 999999;
}

.nav {
    position: fixed;
    top: 0px;
    width: 100%;
    display: flex;
    height: 80px;
    box-shadow: 0 0.1vw 0.5vw 0 #00000066;
    z-index: 99999;
    background-color: #fff;
    align-items: center;
}