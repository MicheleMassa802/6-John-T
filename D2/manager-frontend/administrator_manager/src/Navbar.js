import React from 'react';
import './Navbar.css';

function Navbar() {
  return (
    <div className="navbar">
      <h2 className="navbar-title">Ontario Parks Audio File Manager</h2>
      <div className="navbar-items">
        <a href="/assignment-2-6-3-gaomich4-harishan/" className="navbar-item">Select Park</a>
      </div>
    </div>
  );
}

export default Navbar;
