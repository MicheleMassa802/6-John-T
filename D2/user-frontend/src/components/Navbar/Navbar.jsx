import { Link, Outlet, useNavigate } from "react-router-dom";
import './Navbar.css';
import { useContext, useState, useEffect } from "react";
import logo from '../../assets/logo.png';

const Navbar = () => {

    return (
        <>
            <nav className="nav">
                <Link to="/">
                    <img className="logo-main" src={logo} alt="" />
                </Link>
            </nav>
            <Outlet />
        </>
    )
}

export default Navbar;