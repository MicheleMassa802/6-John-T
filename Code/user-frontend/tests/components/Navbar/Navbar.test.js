import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../../../src/components/Navbar/Navbar.js';
test('should test Navbar component', () => {
 const wrapper = shallow(<Navbar />);
 expect(wrapper).toMatchSnapshot();
});