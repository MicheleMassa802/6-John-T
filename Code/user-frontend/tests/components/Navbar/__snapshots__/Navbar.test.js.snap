// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Navbar component 1`] = `
<Fragment>
  <nav
    className="nav"
    style={
      {
        "align-items": "center",
        "background-color": "#fff",
        "box-shadow": "0 0.1vw 0.5vw 0 #00000066",
        "display": "flex",
        "height": "80px",
        "position": "fixed",
        "top": "0px",
        "width": "100%",
        "z-index": 99999,
      }
    }
  >
    <Link
      to="/"
    >
      <img
        alt=""
        className="logo-main"
        src=""
        style={
          {
            "display": "block",
            "height": "50px",
            "margin-left": "auto",
            "margin-right": "auto",
            "z-index": 999999,
          }
        }
      />
    </Link>
  </nav>
  <Outlet />
</Fragment>
`;
